addons/l10n_ar/static/src/js/tours/account.js

```diff
@@ -10,7 +10,7 @@ odoo.define('l10n_ar.account_tour', function(require) {
     let partner_step_idx = _.findIndex(account_tour.steps, step => step.trigger == 'div[name=partner_id] input');
     account_tour.steps.splice(partner_step_idx + 2, 0, {
         trigger: "div[name=l10n_ar_afip_responsibility_type_id] input",
-        extra_trigger: "[name=move_type][raw-value=out_invoice]",
+        extra_trigger: "[name=move_type] [raw-value=out_invoice], [name=move_type][raw-value=out_invoice]",
```

Comment:
Add the FIXME owl comment:
```
        // FIXME WOWL: this selector needs to work in both legacy and non-legacy views
        // because account_invoice_extracts *adds* a js_class on the base view which forces
        // the use of a legacy view in enterprise only
```

Reply:
added a comment